tests/test_E2_economics.py

```diff
@@ -18,8 +18,11 @@
 }
 
 
-def test_all_cost_info_parameters_added_to_dict_values():
+def test_all_cost_info_parameters_added_to_dict_asset():
+    """
 
+    Tests whether the function get_costs is adding all the calculated costs to dict_asset
+    """
```

Comment:
Just a suggestion for docstring format convention by [PEP257](https://www.python.org/dev/peps/pep-0257/) ^^